CMSgov/mint-app
# Upgrade eslint to v8

# NOREF

## Description

- Upgrades `eslint` from 7 to 8
- Upgrades `prettier` from 2 to 3 (this is what caused most of the changes in the files you see)
- Upgrades (or removes, if duplicative) `eslint-plugin-*` packages.
  - I opted not to touch the `eslint-config-*` packages, as those often caused far more breaking changes, and also aren't typically recommended as upgrades from dependabot

## How to test this change

- Ensure that `scripts/dev lint` doesn't cause any changes locally
- Double check changes in the diff view to make sure nothing seems out of place / like an annoying change or default config.

## PR Author Checklist




## PR Reviewer Guidelines

- It's best to pull the branch locally and test it, rather than just looking at the code online!
- When approving a PR, provide a reason _why_ you're approving it
  - e.g. "Approving because I tested it locally and all functionality works as expected"
  - e.g. "Approving because the change is simple and matches the Figma design"
- Don't be afraid to leave comments or ask questions, especially if you don't understand why something was done! (This is often a great time to suggest code comments or documentation updates)
- Check that all code is adequately covered by tests - if it isn't feel free to suggest the addition of tests.
